src/api/impl/ServerBase.hh

```diff
@@ -14,37 +14,38 @@ class ServerBase : public IServer
 {
 public:
     // === user functions ===
-    GSL_SUPPRESS(f.6) Login::Response Login(_UNUSED const Login::Request& /*request*/) noexcept override
+    GSL_SUPPRESS(f .6) Login::Response Login(_UNUSED const Login::Request& /*request*/) noexcept override
     {
         return UnsupportedApi<Login::Response>(Login::Name());
     }
 
-    GSL_SUPPRESS(f.6) Logout::Response Logout(_UNUSED const Logout::Request& /*request*/) noexcept override
+    GSL_SUPPRESS(f .6) Logout::Response Logout(_UNUSED const Logout::Request& /*request*/) noexcept override
     {
         return UnsupportedApi<Logout::Response>(Logout::Name());
     }
 
-    GSL_SUPPRESS(f.6) StartSession::Response StartSession(_UNUSED const StartSession::Request& /*request*/) noexcept override
+    GSL_SUPPRESS(f .6)
```

Comment:
what is doing this, and won't it cause problems?

Reply:
I'll need to see what's doing it. I don't think it's causing problems though. Better to be safe than sorry though.